tools/SetupFlow/DevHome.SetupFlow/Models/CloneRepoTask.cs

```diff
@@ -232,7 +248,28 @@ IAsyncOperation<TaskFinishedState> ISetupTask.Execute()
                 return TaskFinishedState.Failure;
             }
 
+            SummaryScreenInformation = _host.GetService<CloneRepoSummaryInformationViewModel>();

```

Comment:
>>(); [](http://example.com/codeflow?start=92&length=4)

why not just do "as CloneRepoSummaryInformationViewModel" here?

Reply:
Changed.